pkg/master/master.go

```diff
@@ -570,7 +620,7 @@ func (m *Master) thirdpartyapi(group, kind, version string) *apiserver.APIGroupV
 }
 
 // getExperimentalResources returns the resources for extenstions api
-func (m *Master) getExtensionResources(c *Config) map[string]rest.Storage {
+func (m *Master) getExtensionResources(c *Config, autoscalerStorage, autoscalerStatusStorage rest.Storage) map[string]rest.Storage {
```

Comment:
This function is going to grow a ton of parameters if we do it this way. Maybe add them to the config object? Or pass a map[string]map[string]rest.Storage (group -> path -> rest.Storage)?